pkgs/development/python-modules/furl/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "furl";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1v2lakx03d5w8954a39ki44xv5mllnq0a0avhxykv9hrzg0yvjpx";
+  };
+
+  nativeBuildInputs = [ pythonPackages.flake8 ];
```

nativeBuildInputs maps to `setup_requires`, if this is for tests, put it in `checkInputs`
```suggestion
  nativeBuildInputs = [ flake8 ];
```